frontend/src/app/organizations/organizations.component.html

```diff
@@ -1,13 +1,12 @@
 <div class="container">
     <div class="row">
-        <div class="col-lg-12">
-            <a href="{{ getOrganizationCreateViewUrl() }}" class="btn btn-primary btn-sm">Dodaj organizację</a>    
+        <div class="col-lg-12 mb-2">
+            <a [routerLink]="['create']" class="btn btn-primary">Dodaj organizację</a>
```

Comment:
shouldn't it be `organizations/create`?

Reply:
Without `/` at the beginning, RouterLink directive generates link starting at the current level, so it will add `create` segment to the current `organizations` level.